config/crd/kustomization.yaml

```diff
@@ -28,3 +28,5 @@ resources:
 # the following config is for teaching kustomize how to do kustomization for CRDs.
 configurations:
 - kustomizeconfig.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I know it doesn't really matter but can we put the apiVersion and kind to the top of the file, as it is the convention for K8s resources (which customize kind of follow) ?